bootstrap/command/lint

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
-#
-# Copyright (c) 2017 The Bacardi Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-. $BACARDI_PATH/bootstrap/common/path_info.sh
-
-$(bootstrap_command_path)/format --diff
```

Comment:
Can I understand that you deleted the lint command
to replace lint command with tslint?

Reply:
Yeah, this patch disables clang-format for TypeScript.
Instead introduce tslint for TypeScript.